peakina/readers/csv.py

```diff
@@ -80,7 +80,7 @@ def _line_count(filepath_or_buffer: "FilePathOrBuffer") -> int:
 def csv_meta(
     filepath_or_buffer: "FilePathOrBuffer", reader_kwargs: Dict[str, Any]
 ) -> Dict[str, Any]:
-    total_rows = _line_count(filepath_or_buffer)
+    total_rows = _line_count(filepath_or_buffer, reader_kwargs.get("encoding"))
```

Comment:
So this will work only when encoding is explicitly defined, not going through any auto detection step ?
(guess it's fine but need to keep that in mind if any other issue arise)

Reply:
According to the `open` docs: _In text mode, if encoding is not specified the encoding used is platform dependent: locale.getpreferredencoding(False) is called to get the current locale encoding._

So I guess we just let Python do the auto-detect for us ?